### ISSUE TITLE
Add print options

### ISSUE THREAD

**Issue by alejandrod** [navin-mohan/pychromepdf#11]
Right now the options are part of the `ChromePDF._chrome_options` field. 
Probably it should:
- Add `--print-to-pdf-no-header` by default
- Add the possibility of pass in new params

**Comment by madamadami** [comment]
Yes, the possibility to pass options would be a really great feature! I need passing page width and height as option because in my context it isn't possible to do it by CSS rules.

**Comment by navin-mohan** [comment]
That sounds useful. Feel free to submit a PR.
